lib/srv/app/aws/handler.go

```diff
@@ -54,7 +55,8 @@ type signerHandler struct {
 // SignerHandlerConfig is the awsSignerHandler configuration.
 type SignerHandlerConfig struct {
 	// Log is a logger for the handler.
-	Log logrus.FieldLogger
+	LegacyLogger logrus.FieldLogger
+	Log          *slog.Logger
```

Comment:
I think the GoDoc needs moving down next to this field. You could add a `Deprecated:` to the LegacyLogger as well.

Reply:
I was slacking on it because we'll remove it sooner.
But I'll add it :+1: